### PR TITLE
Add Aspaku

### PR DESCRIPTION
**Pull request by Ariloxe**

### CODE REVIEW

**Comment by scrummer** [comment]
Thank you very much for your contribution @Ariloxe. Please have a look at your images, they are very large. You can use https://compresspng.com/ for example to shrink the file size a bit.

Thank you!

BR
~Scrummer

File: minecraft_servers/aspaku/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "server_name": "aspaku",
+  "nice_name": "Aspaku",
+  "direct_ip": "mc.aspaku.com",
+  "server_wildcards": [
+    "%.aspaku.com"
+  ],
+  "social": {
+    "web": "https://aspaku.com/",
```
Comment (scrummer) [inline minecraft_servers/aspaku/manifest.json:9]: Please remove the trailing slash on the website url.

Comment (Ariloxe) [reply minecraft_servers/aspaku/manifest.json:9]: i'll do that, sorry aha ^^

Comment (Ariloxe) [reply minecraft_servers/aspaku/manifest.json]: good !

**Comment by scrummer** [comment]
Thank you very much!

**Pull request merged**